addresses-b.json

```diff
@@ -0,0 +1 @@
+{"Eidas":"0xb8bCaFdEb78Ae5Aa686c5D00616e1561Db0309d4","AlastriaServiceProvider":"0x83D65fAc900dB0A27B45854AC64DA3D25d97864d","AlastriaIdentityIssuer":"0xE8Dc811f7D05c44EfDD998FFfC26D610B0E5471e","AlastriaCredentialRegistry":"0x2f384716e73ae165bC3427Fd60550c26e6CB91c6","AlastriaPresentationRegistry":"0x95bC1119b764Bd678B68A1c03Cd8c06EfA27Df81","AlastriaPublicKeyRegistry":"0x19e33b2424c1D7B5E563E0A74d8b10CE3812469C","AlastriaIdentityManager":"0xE9B9a6E7700672DBbCe59664457D7E6F1dc00F3c","AlastriaNameService":"0xC1b1Fc2434D50966c28C8573AbdF3c7842a2Abb4"}
```

Comment:
Indent this file to make it easier to read.

Reply:
I think it is generated from fs.write, maybe we need to update the write format